WordPress/Classes/ViewRelated/Post/PostEditorNavigationBarManager.swift

```diff
@@ -73,6 +75,24 @@ class PostEditorNavigationBarManager {
         return button
     }()
 
+    /// Preview Generating Button
+    ///
+    private lazy var previewGeneratingView: LoadingStatusView = {
+        let view = LoadingStatusView(title: NSLocalizedString("Generating Preview", comment: "Message to indicate progress of generating preview"))
+        view.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
⚠️  This looks unnecessary since `LoadingStatusView` already does this in its initializer.